README.md

```diff
@@ -1,4 +1,4 @@
-# [Deploy an Azure VM with RHEL 8_3, IBM HTTP Server ND Traditional V9.0.5 & IBM JDK 8.0 pre-installed](/ihs)
+# [Deploy an Azure VM with RHEL 8_3, IBM HTTP Server V9.0 & IBM JDK 8.0 pre-installed](/ihs)
```

Comment:
Can we change `RHEL 8_3` to `RHEL 8.3`?
Also, is this the latest RHEL version available in Azure?

Reply:
The reason why `8_3` instead of `8.3` is used is that `8_3` is the sku of rhel we currently used to build the base image, see https://github.com/WASdev/azure.websphere-traditional.image/blob/master/ihs/src/main/arm/mainTemplate.json#L228.

`8_3` was the latest RHEL version available in Azure when we were developing base image, pls run `az vm image list --publisher redhat --all -o table` to check available RHEL image versions.